rfcs/meeting-facillitation.md

```diff
@@ -0,0 +1,27 @@
+# Meeting Facillitation
+
+This document aims to formalize our meeting structure with respect to running them.
+
+-----
+
+Each meeting will have a facillitator and a note taker who sign up for the role for each meeting
+ahead of time. These roles are open to all Community Team members. The responsibilities for each are
+listed below.
+
+### Facillitator
+
+- On the Monday before a team meeting an agenda issue will be issued.
+  - This is done by copying the previous weeks agenda and removing unnecessary bits and then opening
+    up the issue
+  - The issue is then posted to IRC on #rust-community and mention all the team leads so they know
```

Comment:
`and all team leads are mentioned`

Reply:
Mention that team leads should fill in their respective topics